Resources/public/js/views/actions/ez-translateactionview.js

```diff
@@ -0,0 +1,136 @@
+/*
+ * Copyright (C) eZ Systems AS. All rights reserved.
+ * For full copyright and license information view LICENSE file distributed with this source code.
+ */
+YUI.add('ez-translateactionview', function (Y) {
+    'use strict';
+    /**
+     * Provides the translate action view class
+     *
+     * @module ez-translateactionview
+     */
+    Y.namespace('eZ');
+
+    /**
+     * Translate Action View
+     *
+     * @namespace eZ
+     * @class TranslateActionView
+     * @constructor
+     * @extends eZ.ButtonActionView
+     */
+    Y.eZ.TranslateActionView = Y.Base.create('translateActionView', Y.eZ.ButtonActionView, [Y.eZ.Expandable], {
+        initializer: function () {
+            this.after({
+                'translateAction': this._toggleExpanded,
+                'expandedChange': this._setClickOutsideEventHandler,
+            });
+        },
+
+        /**
+         * Renders the action
+         *
+         * @method render
+         * @return Y.eZ.TranslateActionView the view itself
+         */
+        render: function () {
+            var container = this.get('container');
+
+            this._addButtonActionViewClassName();
+
+            container.setHTML(this.template({
+                actionId: this.get('actionId'),
+                disabled: this.get('disabled'),
+                label: this.get('label'),
+                hint: this._getTranslateButtonHint(),
```

Comment:
to keep this view consistent with the others button action view, the `hint` attribute should be still be used and you should add a getter function to this attribute to generate the expected text.